examples/hello-world-service/hello_world_service.description

```diff
@@ -1,2 +0,0 @@
-// Let's write a simple HTTP service in Ballerina. 
```

Comment:
Can we remove these now? Shall we remove these in a single PR separately?

Reply:
It would be easier to remove it via this effort also as the Shell fix can also be done now and merged together with this. @mohanvive @DamithaSenevirathne @keizer619 @anupama-pathirage

Reply:
I am also prefer to remove the description files separately in a single PR. We'll update when Shell moved away from referring description files.

Reply:
Created [an issue](https://github.com/ballerina-platform/ballerina-dev-website/issues/4633) to track and remove the `.description` files later in a single PR.